src/zone.js

```diff
@@ -429,6 +430,116 @@ Zone.prototype.createDisk = function(name, config, callback) {
   );
 };
 
+/**
+ * Create an instance group manager in this zone.
+ *
+ * @see [InstanceGroupManager Resource]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers#resource}
+ * @see [InstanceGroupManagers: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers/insert}
+ *
+ * @param {string} name - Name of the instance group manger.
+ * @param {InstanceTemplate} instanceTemplate - Instance template to use for this instance group manager.
```

Comment:
Let's go with a configuration object here to avoid the multiple params. Maybe:

```js
zone.createInstanceGroupManager('name', {
  instanceTemplate: /* string (treated as URL) OR InstanceTemplate (URL created from name) */,
  // ... any other options of an InstanceGroupManager resource
}, callback)
```